app/javascript/components/App.css

```diff
@@ -7,3 +7,15 @@
   display: flex;
   justify-content: center;
 }
+
+#home-header {
+  font-size: 100px;
+  text-align: center;
+}
+
+#play-image {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+}
```

Comment:
Not going to block, but if you haven't already done so, please enable "End of Line" settings in VSCode.